src/main/java/com/github/onsdigital/babbage/search/external/SearchClientExecutorService.java

```diff
@@ -0,0 +1,44 @@
+package com.github.onsdigital.babbage.search.external;
+
+import com.github.onsdigital.babbage.configuration.Configuration;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class SearchClientExecutorService {
+
+    private ExecutorService executorService;
+
+    private static SearchClientExecutorService INSTANCE = new SearchClientExecutorService();
+
+    public static SearchClientExecutorService getInstance() {
+        return INSTANCE;
+    }
+
+    private SearchClientExecutorService() {
+        executorService = Executors.newFixedThreadPool(Configuration.SEARCH_SERVICE.SEARCH_NUM_EXECUTORS);
+        Runtime.getRuntime().addShutdownHook(new Shutdown(this.executorService));
+    }
```

Comment:
This probably should be thread safe. See the double locking example https://dzone.com/articles/all-about-the-singleton

I would also consider making the executor service a static instance and initialising it in the get instance method as well.